### PR TITLE
Pa02

### PR DESCRIPTION
**Pull request by anderhos**
tried my best

### CODE REVIEW

File: src/andershoest_ex/pa02/chutes_simulations.py [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+        """
+        Returning a dictionary mapping player types to lists of game durations
+        a dictionary of lists with the player type as key and the game
+        durations for that specific type as a list of values
+
+        """
+        pass
+
+
+if __name__ == "__main__":
```
Comment (ahmedalbuni) [inline src/andershoest_ex/pa02/chutes_simulations.py:260]: pep8: too many blank lines

File: src/andershoest_ex/pa02/chutes_simulations.py [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+    sim = Simulation(player_field=[Player, LazyPlayer, ResilientPlayer])
+    sim.run_simulation(1)
+    results = sim.result
+    win_per_type = sim.winners_per_type()
+    win_list = [winner[1] for winner in sim.result]
+    board = Board()
+    player = Player(board)
+    #player.position = board.snake_dict[24]
```
Comment (ahmedalbuni) [inline src/andershoest_ex/pa02/chutes_simulations.py]: pep8 leave a space after #

File: src/andershoest_ex/pa02/chutes_simulations.py [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+    player_temp_pos = 24
+    print(player.adjustment)
+
```
Comment (ahmedalbuni) [inline src/andershoest_ex/pa02/chutes_simulations.py]: pep8 extra blank line at the end of the file

File: src/andershoest_ex/pa02/chutes_simulations.py [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+    def __init__(self, board, extra_steps=5):
+        """
+        Initializes ResilientPlayer as Subclass of Player.
+
+        :param: board, extra_steps
+        """
+        super().__init__(board)
+        self.extra_steps = extra_steps
+
```
Comment (ahmedalbuni) [inline src/andershoest_ex/pa02/chutes_simulations.py]: you will need a flag here to know if the player went down a chute, so the next time he should take extra steps

**Review by ahmedalbuni — CHANGES_REQUESTED** [review]
Good job, please fix the pep8 violations and resubmit within a week

File: src/andershoest_ex/pa02/chutes_simulations.py (hunk 1 above)
Comment (anderhos) [reply src/andershoest_ex/pa02/chutes_simulations.py:260]: Fixed

File: src/andershoest_ex/pa02/chutes_simulations.py (hunk 3 above)
Comment (anderhos) [reply src/andershoest_ex/pa02/chutes_simulations.py]: fixed

File: src/andershoest_ex/pa02/chutes_simulations.py (hunk 2 above)
Comment (anderhos) [reply src/andershoest_ex/pa02/chutes_simulations.py]: fixed

**Comment by anderhos** [comment]
Fixed the PEP 8 mistakes.  I also added a few updates.

File: src/andershoest_ex/pa02/chutes_simulations.py [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+    def move(self):    # AH note. Swich move with extra_steps?
```
Comment (ahmedalbuni) [inline src/andershoest_ex/pa02/chutes_simulations.py:110]: You need to update the number of steps here inside the move function

File: src/andershoest_ex/pa02/chutes_simulations.py [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+        used to win the game and the type of the winner.
+
+        Using code from pa01
+
+        :param:
+
+        Making a list of player instances for the different player types.
+
+        """
+
```
Comment (ahmedalbuni) [inline src/andershoest_ex/pa02/chutes_simulations.py]: you need to randomize the players in case randomize players = true

**Review by ahmedalbuni — APPROVED** [review]
Good job!

**Pull request merged**